### PR TITLE
Changed all the download notification from Success to info

### LINKED ISSUE #8593: Change all the notifications related to downloading something from success to info
### Current Behavior
The notifications related to downloading something are not INFO.

### Expected Behavior
The notifications related to downloading something should be INFO.

### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Rajdip019**
**Notes for Reviewers**
Changed all the download notification from Success to info

This PR fixes #8593 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Rajdip019** [comment]
@aabidsofi19 can you look into this?

File: ui/components/MesheryPatterns.js [hunk 1]
```diff
@@ -880,7 +880,7 @@ function MesheryPatterns({
     try {
       downloadFile({ id, name, type : "pattern" })
       updateProgress({ showProgress : false });
-      notify({ message : `"${name}" Design Downloaded` , event_type : EVENT_TYPES.SUCCESS })
+      notify({ message : `"${name}" Design Downloaded` , event_type : EVENT_TYPES.INFO })
```
Comment (leecalcote) [inline ui/components/MesheryPatterns.js:883]: @Rajdip019 let's be consistent with capitalization or lack thereof across messages.

Comment (leecalcote) [inline ui/components/MesheryPatterns.js:883]:
```suggestion
      notify({ message : `"${name}" design downloaded` , event_type : EVENT_TYPES.INFO })
```

**Review by leecalcote — APPROVED** [review]

**Review by aabidsofi19 — APPROVED** [review]
LGTM

File: ui/components/MesheryPatterns.js (hunk 1 above)
Comment (Rajdip019) [reply ui/components/MesheryPatterns.js]: Sorry missed it. Thanks for mentioning. I will try to be more attentive from now on.

**Pull request merged**